_posts/2022-12-21-python-shorts-opensgid-pandas.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Python Shorts: Loading an Open SGID Layer into pandas"
+author:
+  display_name: Jake Adams and Scott Davis
+  email: jdadams@utah.gov
+date: 2022-12-22 11:00:00
+categories:
+  - Featured
+  - Developer
+tags:
+  - python
+---
+
+There are a [myriad of ways]({% link sgid/open-sgid/index.md %}) you can access data from the Open SGID. However, these all require some form of desktop client application. What if you want programmatic access to our spatial data? Because the Open SGID uses the open-source PostGIS extension to Postgres, you can use any libraries that can load Postgres tables and process the geometry data (or just ignore that if all you want is the tabular data).
+
+In the python world, pandas DataFrames are a common and powerful structure for evaluating data. Our developer team has put together this code snippet for loading an SGID layer into either a geopandas geodataframe or an Esri spatially-enabled dataframe.
+
+Now that you've got the SGID layer loaded into a dataframe, you can integrate it into the rest of your application. Our colleague Erik Neemann has written a pretty in-depth [blog post]({% link _posts/2020-05-28-exploring-the-open-sgid-with-open-source-python-tools.md %}) on harnessing the power of geopandas for GUI-less GIS analysis.
+
+## Requirements
+{: .text-left}
+
+This code requires three additional libraries besides pandas (or four if you want a spatially-enabled dataframe). These are all pip-installable from [PyPi](https://pypi.org), so they can be used in the vast majority of python environments.
```

Comment:
**suggestion**: should we link pip to https://github.com/pypa/get-pip?

Reply:
Hmm, good question. I've never had to go out and get pip—it's always either come with the environment (system python, ArcGIS' python installs, Google cloud functions, etc) or been installed automatically with python when creating a new conda environment. Have you run across times where you've had to install it manually?

Reply:
pip doesn't come by default with python. the new installers have it as a checkbox option to install. I'm fine either way but figured it could help someone

Reply:
Guess I've spent too much time in Conda. I think I'm going to leave it out- I'm not telling people how to pip-install, so I think leaving out installing pip makes sense.